docs/ignoring-files-folders-code.md

```diff
@@ -194,8 +200,15 @@ Semgrep AppSec Platform users can disable rules and rulesets through the Policie
 * [Ignore findings in Semgrep AppSec Platform](/semgrep-code/triage-remediation/#ignore-findings)
 * [Ingore findings through GitHub PR comments](/semgrep-code/triage-remediation/#ignore-findings-through-github-pr-comments)
 
+## Troubleshooting
+
+**For GitLab users**: if you use [the `SAST_EXCLUDED_PATHS` variable](https://docs.gitlab.com/ee/user/application_security/sast/#vulnerability-filters) to specify paths excluded from analysis, you may find that Semgrep doesn't honor these items. This is due to default Semgrep behavior. To explicitly exclude files, you must:
+
+1. Create a `.semgrepignore` file that lists the files you want excludes.
+2. [Update the **Path Ignores** box](#define-ignored-files-and-folders-in-semgrep-appsec-platform) in Semgrep AppSec Platform.
```

Comment:
I'm pretty sure one or the other of these is sufficient (doing both is overkill, since they're additive), so we should probably be explicit about that.